.github/workflows/ci.yml

```diff
@@ -59,7 +58,7 @@ jobs:
         uses: anchore/scan-action@v3
         with:
           image: "${{ env.DOCKER_BUILD_REPOSITORY }}:${{ matrix.image }}-${{ matrix.architecture }}-${{ env.SHORT_SHA }}"
-          severity-cutoff: high
+          severity-cutoff: critical
```

Comment:
This is a [CVE-2018-20225](https://nvd.nist.gov/vuln/detail/CVE-2018-20225) related to `pip`. Spun off an issue to address.